androidApp/build.gradle.kts

```diff
@@ -25,11 +25,23 @@ android {
         jvmTarget = "1.8"
     }
     namespace = "com.jetbrains.androidApp"
+    buildFeatures {
+        compose = true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion = "1.5.4-dev-k1.9.20-RC-1edce5fd625"
```

Comment:
Any good reasons why using a RC here instead of a stable version? 🤔
I just ask because the repo is named "basic-example" and therfore I think it shouldn't use any experimental stuff...